x-pack/plugins/apm/server/routes/alerts/rule_types/error_count/register_error_count_rule_type.ts

```diff
@@ -164,7 +167,12 @@ export function registerErrorCountRuleType({
               windowUnit: ruleParams.windowUnit,
             });
 
-            const id = [ApmRuleType.ErrorCount, serviceName, environment]
+            const id = [
+              ApmRuleType.ErrorCount,
+              serviceName,
+              environment,
+              ruleParams.errorGroupingKey,
```

Comment:
@kpatticha do we need `ruleParams.errorGroupingKey` to be added in alert ID? It will always be same value for a given rule as it is coming from the filter. Is it ok if we keep [this implementation](https://github.com/elastic/kibana/pull/155001/files#diff-ecc85d612f224eecc8d0738175891e230652c60cdbcd69630ba2bbe59ccc2856R205)?